.github/workflows/apidiff.yml

```diff
@@ -6,11 +6,15 @@ on:
   pull_request:
 
 jobs:
+  check_docs_only:
+    uses: NikhilSharmaWe/kubebuilder/.github/workflows/check_docs_only.yml@checkDocsOnly
```

Comment:
We cannot merge it with `NikhilSharmaWe/kubebuilder`
Also, it shows a path valid only in your local env/machine